src/pseudopeople/interface.py

```diff
@@ -0,0 +1,35 @@
+import sys
+from pathlib import Path
+from typing import Union
+
+import pandas as pd
+
+from pseudopeople.entities import Form
+from pseudopeople.noise import noise_form
+from pseudopeople.utilities import get_default_configuration
+
+
+def generate_decennial_census(path: Union[Path, str]):
```

Comment:
Please include `year` and `seed` as parameters for this method, to select the year of the decennial census to generate and seed the pseudo-random number generator, respectively.

If possible, I would also like to exclude the `path` parameter, and hide this detail from users until necessary, for a smoother user experience.

Reply:
Adding year and seed is out of scope for this ticket. Regarding the path parameter, I'm not sure how `pseudopeople` would be able to know where to look for the data unless this is provided.

Reply:
> Adding year is out of scope for this ticket. Regarding the path parameter, I'm not sure how `pseudopeople` would be able to know where to look for the data unless this is provided.

Understood re year, I'm concerned that the researchers didn't make it clear enough in the requirements that we want it.  Can we include it as a "TODO" comment, to make sure we don't forget about it?

Re `path`, would it be possible to have the package install the small data in a known location and then to make this parameter optional, with the default being the location of the small data that is installed with the package?

Reply:
We can have the path be optional and act in the way you suggest. We'll add that functionality in when we create the sample data.

Reply:
I can add TODOs for the year and adding a default path. I'll add seed as a parameter.